.circleci/config.yml

```diff
@@ -2,1071 +2,16 @@ version: 2.1
 orbs:
   codecov: codecov/codecov@1.1.1
   azure-cli: circleci/azure-cli@1.0.0
+  path-filtering: circleci/path-filtering@0.1.3
```

Comment:
Is this thing really needed? Can't we run the flaky_test workflow always? Because it checks git itself too, it should be a no-op when no tests changed.

Reply:
It adds minimal overhead to the initial start of the run but it also saves spinning up 32 containers (depending on the parallelism pipeline parameter) for the flaky-test workflow even if there are no tests changed. Leave it up to you.

Reply:
I just moved flaky-test as a job to build-and-test workflow. So having path filters become useless. I deleted continue-config.